src/backend/proxy.ts

```diff
@@ -33,6 +33,54 @@ export const doProxy: any = () => {
     });
 };
 
+// eslint-disable-next-line
+export const doHentDokumentProxy: any = () => {
```

Comment:
Fancy solution! Would it be easier to move this to the api?

Reply:
Do you mean move it to the backend? Yes we can make another endpoint but then it is duplicated someway and it is also not possible to send frontendFeilmelding in case of error